apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/CopyObjectOptions.cs

```diff
@@ -98,6 +98,18 @@ public sealed class CopyObjectOptions
         /// </summary>
         public Object ExtraMetadata { get; set; }
 
+        /// <summary>
+        /// The encryption key to use for this operation. If this property is null, the <see cref="StorageClient.EncryptionKey"/>
+        /// will be used instead. Use <see cref="EncryptionKey.None"/> to remove encryption headers from this request.
+        /// </summary>
+        public EncryptionKey EncryptionKey { get; set; }
+
+        /// <summary>
+        /// The encryption key to use for the source of the copy. The <see cref="EncryptionKey"/> and
+        /// <see cref="StorageClient.EncryptionKey"/> properties are used for the target of the copy.
+        /// </summary>
+        public EncryptionKey SourceEncryptionKey { get; set; }
```

Comment:
Should this default to using the client's encryption key as well? It seems like a copy should be analogous to a download then upload using the client, but a download then upload wouldn't require specifying a key if the object were already encrypted with the client's key whereas a copy would.